_docs/info/migrate_rpool.md

```diff
@@ -0,0 +1,143 @@
+config:
+
+        NAME        STATE     READ WRITE CKSUM
+        rpool       ONLINE       0     0     0
+          mirror-0  ONLINE       0     0     0
+            c1t3d0  ONLINE       0     0     0
+            c1t4d0  ONLINE       0     0     0
+
+```
+
+## Attach the new disk(s) to rpool
+
+The first step is to attach the new disks to the existing rpool using the command **zpool attach** like this:
+
+``` terminal
+$ sudo zpool attach rpool c1t1d0 c1t3d0
+Password: 
+Make sure to wait until resilver is done before rebooting.
+```
+
+Running the command **zpool status** after attaching the new disk will give this output:
+
+``` terminal
+$ zpool status
+  pool: rpool
+ state: ONLINE
+status: One or more devices is currently being resilvered.  The pool will
+        continue to function, possibly in a degraded state.
+action: Wait for the resilver to complete.
+  scan: resilver in progress since Mon Apr 13 14:24:45 2020
+        18.1M scanned out of 1.60G at 1.21M/s, 0h22m to go
+    16.1M resilvered, 1.10% done
+config:
+
+        NAME        STATE     READ WRITE CKSUM
+        rpool       ONLINE       0     0     0
+          mirror-0  ONLINE       0     0     0
+            c1t1d0  ONLINE       0     0     0
+            c1t2d0  ONLINE       0     0     0
+            c1t3d0  ONLINE       0     0     0
+
+```
+
+**You must not do anything until resilver is done**. So go and grab a cup of coffee or some other fluid you want to drink.
+
+To monitor progress you can use this command:
+
+``` terminal
+$ yes "zpool status; sleep 10"|sh
+
+Use Ctrl+c to quit the command.
+```
+
+
+The above should be done for every new disk which in this case means one more time for c1t4d0
+
+## Install boot block and boot loader to the new disk(s)
+
+The next step is to install the boot block and the boot loader to the new disk(s) using the command **bootadm install-bootloader** like this:
+
+``` terminal
+$ sudo bootadm install-bootloader -P rpool
+```
+
+## Reboot to test migration was successful
+
+Before detaching the old disk(s) from the rpool we make a reboot to ensure everything was successful. **Remember to change the first boot device in the BIOS to use the new disk (or one of the new disk(s)).**
+
+When rebooting you might see a warning from 'devid register' that device ids has changed in case you need to reorder the disks in the BIOS to be able to change boot device. This is just a warning which will be fixed by creating a new boot_archive either manually or automatically by OmniOSce when you reboot or power down your system.
+
+## Detach old disk(s) from the rpool
+
+If you were able to make a successful reboot on the new disk(s) then the time has come to detach the old disk(s) from the rpool using the command **zpool detach** like this:
+
+``` terminal
+sudo zpool detach rpool c1t1d0
+sudo zpool detach rpool c1t2d0
+```
+
+After detaching the old disk(s) from the rpool you should do a reboot to ensure everything is fine.
+
```

Comment:
I would add a note about pool auto resizing if the new disks are larger than the old ones ... 

```
zpool set autoexpand=on rpool
```

Reply:
Added information in the latest commit.